### PR TITLE
Verifying the BigQuery Storage client is closed

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: connector/src/test/java/com/google/cloud/spark/bigquery/v2/BigQueryInputPartitionReaderTest.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ public void testReadAvro() throws Exception {
                         ALL_TYPES_TABLE_AVRO_RAW_SCHEMA
                 );
 
-        BigQueryInputPartitionReader reader = new BigQueryInputPartitionReader(readRowsResponses, converter);
+        BigQueryInputPartitionReader reader = new BigQueryInputPartitionReader(readRowsResponses, converter, null);
```
Comment (Gaurangi94) [inline connector/src/test/java/com/google/cloud/spark/bigquery/v2/BigQueryInputPartitionReaderTest.java:84]: why is this null? Won't it give null pointer exception?

Comment (davidrabinowitz) [reply connector/src/test/java/com/google/cloud/spark/bigquery/v2/BigQueryInputPartitionReaderTest.java:84]: No, as I'm not calling `close()` in the test. The input is mocked anyway

**Pull request merged**